### PR TITLE
Influx API handlers

### PR DESCRIPTION
**Pull request by venkateshamatam**
# Description
Influx API handlers

# Testing
- Tested using postman
# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: pkg/influx/query/query.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+					]
+				}
+			]
+		}`
+		ctx.SetContentType("application/json")
+		_, err := ctx.Write([]byte(databaseResponse))
+		if err != nil {
+			log.Errorf("GetQueryHandler: failed to write response, err=%v", err)
+		}
+	} else if q == "CREATE DATABASE benchmark WITH REPLICATION 1" {
```
Comment (nkunal) [inline pkg/influx/query/query.go:33]: we should just check if `q.contains("CREATE DATABASE")` 
we don't care about what the dbName is and we should just return 200 for this

File: pkg/influx/query/query.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		// Return status code 200
+		ctx.SetStatusCode(fasthttp.StatusOK)
+	} else {
+		ctx.Error("Unsupported query", fasthttp.StatusBadRequest)
+	}
+}
+func PostQueryHandler(ctx *fasthttp.RequestCtx, myid uint64) {
+	q := string(ctx.QueryArgs().Peek("q"))
+	if q == "drop database benchmark" {
```
Comment (nkunal) [inline pkg/influx/query/query.go:42]: same comment as above. also make sure it case insensitive check

File: pkg/influx/query/query.go (hunk 1 above)
Comment (venkateshamatam) [reply pkg/influx/query/query.go]: I've pushed the changes for this to check if the query contains `create database`. Do we want to return an error if it just contains `create database` without a database name?

File: pkg/influx/query/query.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+	log "github.com/sirupsen/logrus"
+	"github.com/valyala/fasthttp"
+)
+
+func GetQueryHandler(ctx *fasthttp.RequestCtx, myid uint64) {
+
+	q := string(ctx.QueryArgs().Peek("q"))
+	if strings.ToLower(q) == "show databases" {
+		databaseResponse := `{
```
Comment (nkunal) [inline pkg/influx/query/query.go:14]: make this as a const at the top of file, so that we dont have to recreate this memory

Comment (nkunal) [reply pkg/influx/query/query.go:14]: name it `INFLUX_SHOW_DB_RESP`

**Review by nkunal — APPROVED** [review]

**Pull request merged**